packages/snaps-sdk/src/types/methods/get-currency-rate.ts

```diff
@@ -0,0 +1,29 @@
+export type Currency = 'btc';
+
+/**
+ * The rate object.
+ *
+ * @property conversionRate - The conversion rate. It maps a currency code (e.g. "btc") to its
+ * conversion rate
+ * @property conversionDate - The date of the conversion rate as a UNIX timestamp.
+ * @property usdConversionRate - The conversion rate to USD.
```

Comment:
What's the difference between this and `conversionRate`?

Reply:
`conversionRate` is the rate for the cyptocurrency in your selected Fiat in extension. `usdConversionRate` is always from USD.

Reply:
Should we include the selected currency then? 😅